.vscode/tasks.json

```diff
@@ -115,66 +77,11 @@
             "group": "none",
             "problemMatcher": ["$gcc"]
         },
-        {
-            "label": "Run Setup Payload Tests",
-            "type": "shell",
-            "command": "scripts/tests/setup_payload_tests.sh",
-            "group": "none",
-            "dependsOn": "Bootstrap",
-            "problemMatcher": ["$gcc"]
-        },
-        {
-            "label": "Build QRCode Payload Tests",
-            "type": "shell",
-            "command": "scripts/tests/qrcode_payload_tests.sh",
-            "group": "none",
-            "dependsOn": "Bootstrap",
-            "problemMatcher": ["$gcc"]
-        },
-        {
-            "label": "Build & Run Crypto Tests",
```

Comment:
This is removing a bunch of tasks.  Should we convert them to running on top of the GN build instead?

Reply:
I actually did convert them initially, although I was uncertain if we still needed them - they are redundant as long as it's fast enough to run the whole suite, which I'm hopeful is the case now.

Then I noticed a typo:  scripts/tests/crypt_tests.sh

So I guess nobody is using this, because it clearly doesn't work..

Happy to add them back if they are useful, I'm just not sure if they are.

Reply:
these were intended to salve the pain of waiting for a recursive make to complete when iterating (change line, compile, test for crypto).  builds with gn and ninja are fast enough that "build and test all" is viable iteration target.